.packit.yaml

```diff
@@ -0,0 +1,39 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: rubygem-foreman-tasks.spec
+
+# add or remove files that should be synced
+files_to_sync:
+    - rubygem-foreman-tasks.spec
+    - foreman-tasks.logrotate
+    - .packit.yaml
+
+# name in upstream package repository or registry (e.g. in PyPI)
+upstream_package_name: foreman-tasks
+# downstream (Fedora) RPM package name
+downstream_package_name: rubygem-foreman-tasks
+
+actions:
+  post-upstream-clone:
+    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/develop/packages/plugins/rubygem-foreman-tasks/rubygem-foreman-tasks.spec -O rubygem-foreman-tasks.spec"
+    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/develop/packages/plugins/rubygem-foreman-tasks/foreman-tasks.logrotate -O foreman-tasks.logrotate"
+  get-current-version: "ruby -I lib/ -r foreman_tasks/version -e 'puts ForemanTasks::VERSION'"
+  create-archive:
+    - gem build foreman-tasks.gemspec
+    - bash -c "ls -1t ./foreman-tasks-*.gem | head -n 1"
+
+jobs:
+  - job: copr_build
+    trigger: pull_request
+    targets:
+      centos-stream-8:
+        additional_modules: "foreman:el8,ruby:2.7,nodejs:12"
```

Comment:
Keeping this in sync between repos is going to be a pain. Why do we need to list the modules that `foreman:el8` depends on?

Reply:
because `foreman:el8` does not depend on `nodejs:12`

Reply:
(correct for ruby tho)

Reply:
So it doesn't install the ones we specify as build dependencies? It feels like it should so RFE time? :)

Reply:
My understanding is that the build dependencies we specify only apply in the context of "build inside the module", but we're not doing that.

Reply:
It kind of feels like we should. The target package is built inside the module, and we want to replicate that. Essentially I'm looking for the equivalent of `BuildRequires` for modularity. So something like `foreman:el8;with-build`. It feels like we're once again hitting an area of modularity where few have gone before.

Reply:
No, the target package is not built *inside* a module. It's built with modules enabled in the buildroot.

If we'd want to have those build *inside* a module we'd need MBS, which we neither have nor want.

Reply:
I meant that we want to replicate the build environment in terms of packages available.